[Bug]: Using Checkbox crashes build

### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-angular/blob/main/docs/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-angular/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Functionality

### It affects the following packages

Components

### Which version of Lyne Components are you using

19.0.0-beta.3

### Operating system

_No response_

### Browser / Browser version

_No response_

### Input mode

None

### Additional settings

_No response_

### What happened?

When using the checkbox component the build crashes:
```
✘ [ERROR] Could not resolve "@angular/cdk/a11y"

    node_modules/@sbb-esta/lyne-angular/fesm2022/sbb-esta-lyne-angular-checkbox-checkbox.mjs:1:29:
      1 │ import { FocusMonitor } from '@angular/cdk/a11y';
        ╵                              ~~~~~~~~~~~~~~~~~~~
```

### Steps to reproduce the issue

1. Open Stackblitz https://stackblitz.com/edit/checkbox-crashes-build?file=src%2Fmain.ts
2. Look at Terminal output


### Relevant log output

```shell

```

--- a/src/showcase/docs/getting-started.mdx
+++ b/src/showcase/docs/getting-started.mdx
@@ -10,16 +10,16 @@ import { Meta } from '@storybook/blocks';
 # Getting Started
 
 1.  Install Angular CLI, see [Angular CLI documentation](https://cli.angular.io/)
-2.  Install the `@sbb-esta/lyne-angular` and the `@sbb-esta/lyne-elements` packages:
+2.  Install the `@sbb-esta/lyne-angular`, `@sbb-esta/lyne-elements` and `@angular/cdk` packages:
 
     ```sh
-      npm install --save @sbb-esta/lyne-angular @sbb-esta/lyne-elements
+      npm install --save @sbb-esta/lyne-angular @sbb-esta/lyne-elements @angular/cdk
     ```
 
     or, if using yarn:
 
     ```sh
-      yarn add @sbb-esta/lyne-angular @sbb-esta/lyne-elements
+      yarn add @sbb-esta/lyne-angular @sbb-esta/lyne-elements @angular/cdk
     ```
 
 3.  Including global styles is strongly recommended to apply all SBB styles to your application.

